packages/rocketchat-lib/server/functions/sendMessage.js

```diff
@@ -1,10 +1,66 @@
-import _ from 'underscore';
+const validateAttachmentsFields = attachmentFields => {
```

Comment:
@ggazzo why did you change the code style from parenthesis around the arguments to not?

Reply:
because we dont need `( )`in this case :)